client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -36,13 +44,132 @@ const UserPage = createClass({
+		const sortedBrews = this.sortBrews(brews, this.state.sortType);
 
 		return _.map(sortedBrews, (brew, idx)=>{
 			return <BrewItem brew={brew} key={idx}/>;
 		});
 	},
 
+	sortBrews : function(brews, sortType){
+		if(sortType == 'alpha') {
+			return _.orderBy(brews, (brew)=>{ return brew.title; }, this.state.sortDir);
+		}
+		if(sortType == 'created'){
+			return _.orderBy(brews, (brew)=>{ return brew.createdAt; }, this.state.sortDir);
+		}
+		if(sortType == 'updated'){
+			return _.orderBy(brews, (brew)=>{ return brew.updatedAt; }, this.state.sortDir);
+		}
+		if(sortType == 'views'){
+			return _.orderBy(brews, (brew)=>{ return brew.views; }, this.state.sortDir);
+		}
+		if(sortType == 'latest'){
+			return _.orderBy(brews, (brew)=>{ return brew.lastViewed; }, this.state.sortDir);
+		}
+		return _.orderBy(brews, (brew)=>{ return brew.title; }, this.state.sortDir);
+	},
+
+	handleSortOptionChange : function(event){
+		this.setState({
+			sortType : event.target.value
+		});
+	},
+
+	handleSortDirChange : function(event){
+		const newDir = (this.state.sortDir == 'asc' ? 'desc' : 'asc');
+		this.setState({
+			sortDir : `${newDir}`
+		});
+	},
+
+	renderSortOptions : function(){
+		return <div className='sort-container'>
+			<table>
+				<tr>
+					<td>
+						<h6>Sort Type :</h6>
+					</td>
+					<td>
+		  		<label>
+							<input
+								type='radio'
+								name='sortType'
+								value='alpha'
+								checked={this.state.sortType == 'alpha'}
+								className='sort-radio'
+								onChange={this.handleSortOptionChange}
+							/>
+			Title
+						</label>
+					</td>
+					<td>
+						<label>
+							<input
+								type='radio'
+								name='sortType'
+								value='created'
+								checked={this.state.sortType == 'created'}
+								className='sort-radio'
+								onChange={this.handleSortOptionChange}
+							/>
+			Created Date
+						</label>
+					</td>
+					<td>
+						<label>
+							<input
+								type='radio'
+								name='sortType'
+								value='updated'
+								checked={this.state.sortType == 'updated'}
+								className='sort-radio'
+								onChange={this.handleSortOptionChange}
+							/>
+			Updated Date
+		  		</label>
+				  </td>
+				  <td>
+				  <label>
+							<input
+								type='radio'
+								name='sortType'
+								value='views'
+								checked={this.state.sortType == 'views'}
+								className='sort-radio'
+								onChange={this.handleSortOptionChange}
+							/>
+			Views
+		  		</label>
+				  </td>
+				  <td>
+				  <label>
+							<input
+								type='radio'
+								name='sortType'
+								value='latest'
+								checked={this.state.sortType == 'latest'}
+								className='sort-radio'
+								onChange={this.handleSortOptionChange}
+							/>
+			Latest
+		  		</label>
+				  </td>
+				  <td>
+						<h6>Direction :</h6>
+					</td>
+					<td>
+						<button
+							onClick={this.handleSortDirChange}
+						>
+							{`${(this.state.sortDir == 'asc' ? '⮝ ASC' : '⮟ DESC')}`}
```

Comment:
Are those unicode characters there? Can we write them with `\uXXXX` format?
![image](https://user-images.githubusercontent.com/2605467/126964314-6320e005-094d-47f8-b502-77e0c94a8c31.png)

Reply:
Should be done now.